### ISSUE TITLE
Error with Gradle

### ISSUE THREAD

**Issue by carmas123** [mcxiaoke/kotlin-koi#2]
If I try to use (like documented)

```
compile 'com.mcxiaoke.koi:core:0.5.1'
compile 'com.mcxiaoke.koi:async:0.5.1'
```

I got this error:

Error:Module 'com.mcxiaoke.koi:core:0.5.1' depends on one or more Android Libraries but is a jar

**Comment by carmas123** [comment]
solved with this:

```
compile('com.mcxiaoke.koi:core:0.5.1') {
    exclude group: 'com.android.support', module: 'support-v4'
}
compile('com.mcxiaoke.koi:async:0.5.1') {
    exclude group: 'com.android.support', module: 'support-v4'
}
```

**Comment by mcxiaoke** [comment]
thanks, I'll fix it in next release.

**Comment by mcxiaoke** [comment]
fixed in 0.5.2
